src/utils/ga.ts

```diff
@@ -17,13 +17,12 @@ export const LogActions = Object.freeze({
     Interval: 'Interval',
     Template: 'Template',
 });
-
 /* eslint-disable */
 export function initGA() {
     /* @START-EXCLUDE: 'lib' */
-    if (!window.isProductionWebsite) return;
-    (function (i, s, o, g, r, a, m) {
-        i.GoogleAnalyticsObject = r;
+    if (!(window as any).isProductionWebsite) return;
```

Comment:
```suggestion
    if (!window.isProductionWebsite) return;
```
please consider this change for all 3 occurrences in this file: at line 23, 42 & 48 😊